docs/host-parameters-validation.md

```diff
@@ -0,0 +1,19 @@
+# HostParametersValidation
+
+## Description
+
+This rule is to valiate the parameters in the 'x-ms-parameterized-host' to ensure they are following below rules:
+
+1. If a parameter matches belows, therefore it must be called 'endpoint' and be typed 'type:string, format:uri'.
```

Comment:
If a parameter matches one of the following cases, it must be defined as 'endpoint' and must be claimed with 'type:string, format:uri' explicitly.

Reply:
"it must be defined as 'endpoint'"  seems a little confusion, here it mean the parameter name must be 'endpoint'